docs/topic-guides/sessions/session-basics.rst

```diff
@@ -109,6 +146,67 @@ configuration options.
 |                              | lot of data. Refer to :ref:`Data in Renku <data>` for further information                 |
 +------------------------------+-------------------------------------------------------------------------------------------+
 
+.. _create_autostart_links_anchor:
+
+Create autostart links
+~~~~~~~~~~~~~~~~~~~~~~
+
+Autostart links simplify starting a session and landing there without requiring any further
+clicking on the UI. Unless you change options, the autostart link will pick the default branch
+on the target project; if any session is already running, the user lands there. Otherwise, a
+session is started either from the previously unsaved work (if any) or the latest commit.
```

Comment:
```suggestion
Autostart links are a good way to give others access to sessions, since they bring the visitor directly into a session without having to search for projects or provide any information. 

The standard behavior is for the autostart link to use the default branch
on the target project; if any session is already running, the user is taken there there. Otherwise, a
session is started either from the previously unsaved work (if any) or the latest commit.
```